lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -0,0 +1,127 @@
+---
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | inactive | resolved
+  url: http://hl7.org/fhir/allergyintolerance-definitions.html#AllergyIntolerance.clinicalStatus
```

Comment:
I think all the urls in this should include r4.

[See Encounter for an example](https://github.com/cerner/fhir.cerner.com/blob/1cc663359bea6bd58e5fef5d46e47e06745dbb0b/lib/resources/r4/encounter.yaml)

Reply:
Updated links: https://github.com/cerner/fhir.cerner.com/pull/259/commits/45bf50060a45dbac596c843e5c9995bce8e1b1da